scripts/postinstall.sh

```diff
@@ -3,3 +3,6 @@
 set -e
 
 cp node_modules/testpilot-metrics/testpilot-metrics.js src/vendor
+cp node_modules/kinto-http/dist/kinto-http.js src/sidebar/vendor
```

Comment:
Do you want the non-minimized _kinto-http.js (1.0M)_ vs _kinto-http.**min**.js (230K)_?

```sh
➜  notes git:(master) ✗ ls -lash node_modules/kinto-http/dist

2024 -rw-r--r--   1 pdehaan  staff   1.0M May  9 15:29 kinto-http.js
 464 -rw-r--r--   1 pdehaan  staff   230K May  9 15:29 kinto-http.min.js
 144 -rw-r--r--   1 pdehaan  staff    70K May  9 15:29 kinto-http.noshim.js
 200 -rw-r--r--   1 pdehaan  staff    99K May  9 15:29 moz-kinto-http-client.js
```